bigbluebutton-html5/imports/ui/components/audio/echo-test/component.jsx

```diff
@@ -54,10 +54,9 @@ class EchoTest extends Component {
     } = this.props;
     const disableYesButtonClicked = callback => () => {
       this.setState({ disabled: true }, callback);
-      let  checknumber = sessionStorage.getItem(AUDIO_TEST_NUM_KEY);
-      checknumber = true;
-      sessionStorage.setItem(AUDIO_TEST_NUM_KEY, checknumber);
-      return checknumber;
+      audioTestPassed = true;
+      sessionStorage.setItem(AUDIO_TEST_NUM_KEY, audioTestPassed);
+      return audioTestPassed;
```

Comment:
When you changes everywhere to audioTestPassed, should you not change AUDIO_TEST_NUM_KEY to AUDIO_TEST_PASSED_KEY?